### ISSUE TITLE
[Feature Request] Option to add Video files to sort

### ISSUE THREAD

**Issue by Ltek** [acrojax-open/photos2folders#1]
I just found this app - EXCELLENT WORK!  I'm hoping you'd be kind and add common video files (MTS, MP4, etc) to its capabilities. I like to keep photos and videos in the same folders. thx!

**Comment by beer2279** [comment]
Agreed very good app.  same comment as Ltek just need to keep standard video files organized in the same folders.  i had to go back through many folders to move the videos.

**Comment by borgib** [comment]
I agree with the others. The application is perfect for my needs with this one missing feature. I use this to organize the thousands of photos taken with our smartphones, but it leaves the video files behind :-(
